### PR TITLE
[Enhancements] Employee page changes closes #80

### LINKED ISSUE #80: [Bug] Allow null selection/display on dept/loc/emp select fields

### PR DESCRIPTION
**Pull request by jscottchapman**
Styled contact buttons.
Created the ability to select no supervisor, no department, and no
location.
Made the page tolerant of very little information being entered for the
employee.

### CODE REVIEW

File: app/styles/granite/global/utility.less [hunk 1]
```diff
@@ -46,17 +46,6 @@
...
-  box-shadow: 0 3px 6px fade(@granite-grey, 70%);
-  z-index: 1;
-  transition: all ease-in-out 0.35s;
-
-  &:hover {
-    background-color: lighten(@granite-green, 15%);
-    box-shadow: 0 3px 7px fade(@granite-grey, 65%);
-  }
-}
-
```
Comment (James1x0) [inline app/styles/granite/global/utility.less]: 👍

File: app/controllers/account/employee/index/index.js [hunk 2]
```diff
@@ -2,7 +2,18 @@ import Ember from 'ember';
 const { computed, Controller } = Ember;
 export default Controller.extend({
```
Comment (James1x0) [inline app/controllers/account/employee/index/index.js]: 👾 Remove extra line here

File: app/templates/account/employee/index/index.hbs [hunk 3]
```diff
@@ -45,50 +45,74 @@
...
-          Date of Hire: {{time model.hireDate}} <br />
-          Birthday: {{time model.dateOfBirth}}
-        </p>
+        {{#if hasAnniversaries}}
+          <p class="text">
+            {{#if model.hireDate}}
+              Date of Hire: {{time model.hireDate}} <br />
+            {{/if}}
+            {{#if model.dateOfBirth}}
+              Birthday: {{time model.dateOfBirth}}
```
Comment (James1x0) [inline app/templates/account/employee/index/index.hbs]: Can you change this to `Date of Birth`? Seems to align more with the model's data key and the date of hire label.

**Comment by jscottchapman** [comment]
Should be good to go now.  💫

**Comment by James1x0** [comment]
🎉

**Pull request merged**